pack/ant_encounter.json

```diff
@@ -12,7 +12,7 @@
 		"position": 25,
 		"quantity": 1,
 		"spoiler": 1,
-		"text": "<b>Give to the Scott Lang player.</b><br>\nYou may flip to alter-ego form. Choose:\n·Exhaust Scott Lang → remove Care for Cassie form the game.<br>\n·Choose and discard 1 card from your hand. You cannot change form until your next turn ends. Discard this obligation.",
+		"text": "<b><i>Give to the Scott Lang player.</i></b><br>\nYou may flip to alter-ego form. Choose:\n- Exhaust Scott Lang → remove Care for Cassie form the game.<br>\n- Choose and discard 1 card from your hand. You cannot change form until your next turn ends. Discard this obligation.",
```

Comment:
No strong feelings on my part, but is there a reason to use  hyphen instead of a middot? From what I can tell, the UI in the app does not do anything special with either.

Reply:
One other thing I'm seeing, we could replace the `<b><i>` wrappers with `[[ ]]`, right? Doesn't look like there are any `<i><b>`, luckily. 😆

Reply:
The mid-dot just looks quite bad compared to the hyphen, take for example [Ant Man's obligation](https://marvelcdb.com/card/12025) vs [Widow's bite](https://marvelcdb.com/card/08004). Also, once again, there was no real consistency across the project on how the lists were done, so I ended up sticking with what was already there (the hyphens) which happens to look better imo than those tiny, almost unrecognisable dots.

They should probably be actual bullet points, but in all honesty, I haven't tried the syntax for HTML lists mainly because it will probably make it harder for other people to define:

```html
<ul>
  <li>Exhaust Scott Lang → remove Care for Cassie from the game.
  <li>Choose and discard 1 card from your hand. You cannot change form until your next turn ends. Discard this obligation.
</ul>
```

Which should produce something much closer to how it looks on the card:

* Exhaust Scott Lang → remove Care for Cassie from the game
* Choose and discard 1 card from your hand. You cannot change form until your next turn ends. Discard this obligation